packages/portalnetwork/src/networks/history/types.ts

```diff
@@ -191,10 +191,20 @@ export const HistoricalRootsBlockProof = new ContainerType({
   slot: SlotType,
 })
 
+export const HistoricalSummariesProof = new VectorCompositeType(Bytes32Type, 13)
+
+export const HistoricalSummariesBlockProof = new ContainerType({
```

Comment:
```suggestion
export const HistoricalSummariesWithBlockProof = new ContainerType({
```

Reply:
I'd rather keep these types named as is since they match the naming convention in the [portal network specs](https://github.com/ethereum/portal-network-specs/blob/master/beacon-chain/beacon-network.md#historicalsummaries).  Any objection?